First iteration of the POI view

gmarty opened the pull request:
This patch contains some code related to the distance of the beacon, but it's not yet being used.

Things missing:
* Button to follow/unfollow the channel
* Distance to the beacon
* Embedded map for geotagged beacons
* Suggestions of additional POIs

gmarty committed:
First iteration of the POI view

wilsonpage commented:
- URLs are wrapping
- For simplicity can we just dump the content underneath the image for now.
- I suggest putting the URL directly under the image and putting the button after the description.
- We can polish the styling later, just need it to be functional for now.
- Need to pref the `itemsExpandable` flag on for this to work.

<img src="https://cloud.githubusercontent.com/assets/1020551/19970039/abd55c7a-a1d2-11e6-9826-becfd982c444.png" width=320/>

gmarty commented:
I fixed the issues.
The url can now expand on multiple lines.
I moved the URL underneath the image and the website button under the description.

How does it look now?

wilsonpage commented:
@gmarty looks good!
- Could we change the 'Website' button to 'Visit' or if we have [`siteName`](https://github.com/mozilla-magnet/magnet-html-parser/blob/master/lib/document-parser.js#L94) in the metadata: 'Visit SITE_NAME'.
- Could you make sure all spacing (margin/padding) around URL, Title, Description and Button are equal. For example the left spacing of the URL text is much greater than the top.

wilsonpage commented:
Also needs a rebase.
